.github/workflows/cron_jerigon_zero_testing.yml

```diff
@@ -0,0 +1,102 @@
+name: Jerigon Zero Testing
+
+on:
+  # Uncomment when ready to run on a schedule
+  # schedule:
+  #   # Run every Sunday at 12:00 AM (UTC)
+  #   - cron: "0 0 * * 0"
+  push:
+    branches: [develop]
+  pull_request:
+    branches:
+      - "**"
+  workflow_dispatch:
+    branches:
+      - "**"
+
+env:
+  CARGO_TERM_COLOR: always
+  REGISTRY: ghcr.io
+
+jobs:
+  jerigon_zero_testing:
+    name: Jerigon Zero Testing - Integration and Benchmarking
+    runs-on: zero-ci
+    steps:
+      - name: Checkout zk_evm code
+        uses: actions/checkout@v4
+
+      - name: Setup Rust Toolchain
+        uses: actions-rust-lang/setup-rust-toolchain@v1
+
+      - name: Set up Rust Cache
+        uses: Swatinem/rust-cache@v2
+        with:
+          cache-on-failure: true
+
+      - name: Build the project
+        run: |
+          RUSTFLAGS='-C target-cpu=native -Zlinker-features=-lld -Copt-level=3' cargo build --release
+          sudo sysctl kernel.perf_event_paranoid=0
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v3
+
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@v2
+        with:
+          registry: ${{ env.REGISTRY }}
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Run Erigon Network
+        run: |
+          cd ..
+          tar xf "$(pwd)/zk_evm/test_data/erigon-data.tar.gz" || {
```

Comment:
@praetoriansentry what would be best way to store the file?

Reply:
It is not big, I think it can stay with repo.

Reply:
we are using a shorter data (672) blocks, if we use that, its probably fine but if we choose to go with a larger set of data, we may need to think

Reply:
@Nashtare where should we store the erigon data file? the one with ~6500 blocks zip file is around 115MB. i dont think putting that file might increase the repo size.

Reply:
I think we have options to store this, with IPFS or else.

Reply:
Note that there are some prover killers before block 700, so we may want to test only 700 and onwards. Also for regression check, we can use the testing config Sai merged in #742